### PR TITLE
Check and load HLS video if initial load failed (expando/hidden etc.) + UI bugfixes

### PR DESCRIPTION
**Pull request by MoMitch**
Initialize scrub thumb position on HTML5StreamPlayer scrub start.
Reduce state dependencies in the HTML5StreamPlayer component (state updates occur as needed but current state is retrieved from video tag where possible).
Add additional loading checks incase initial load fails.
Align scrub track to player buttons

### CODE REVIEW

**Comment by MoMitch** [comment]
@birakattack

File: src/app/components/HTML5StreamPlayer/index.jsx [hunk 1]
```diff
@@ -123,19 +130,37 @@ class HTML5StreamPlayer extends React.Component {
...
+      }
 
       const video = this.refs.HTML5StreamPlayerVideo;
-      if ((this.props.postData.videoPlaytime > 0) && (video.paused === false)) {
-        this.setState({totalServedTime: this.props.postData.videoPlaytime * 1000.0});
+      if (this.props.postData.videoPlaytime > 0) {
+        this.setState({videoLoaded: true,
+          totalServedTime: this.props.postData.videoPlaytime * 1000.0,
+          totalTime: this.secondsToMinutes(video.duration),
+          videoWasInView: false,
```
Comment (birakattack) [inline src/app/components/HTML5StreamPlayer/index.jsx]: Can we fix the formatting here this should look like:
```
this.setState({
    videoLoaded: true,
    totalServedTime: ...
});
```

File: src/app/components/HTML5StreamPlayer/index.jsx [hunk 2]
```diff
@@ -123,19 +130,37 @@ class HTML5StreamPlayer extends React.Component {
...
+        });
         this.sendTrackVideoEvent(VIDEO_EVENT.CHANGED_PAGETYPE, this.getPercentServed());
+        video.currentTime = this.props.postData.videoPlaytime;
+      } else {
+        this.setState({videoLoaded: true,
+          totalTime: this.secondsToMinutes(video.duration),
+          videoWasInView: false,
```
Comment (birakattack) [inline src/app/components/HTML5StreamPlayer/index.jsx]: Format like the comment above

File: src/app/components/Post/PostContent/index.jsx [hunk 3]
```diff
@@ -430,7 +430,7 @@ function renderVideo(videoSpec, posterImage, aspectRatio, props) {
     } else {
       aspectRatio = getAspectRatio(false, videoSpec.width, videoSpec.height);
     }
-
+
```
Comment (birakattack) [inline src/app/components/Post/PostContent/index.jsx]: Whitespace, please remove

**Pull request merged**